board_client/animations/cobra_mk3.py

```
@@ -0,0 +1,142 @@
+        zoom_max = base_scale * 1.35
+    else:
+        # Default: use min(w, h) as before
+        base_scale = min(w, h) / 220
+        xoff = w // 2
+        yoff = h // 2
+        zoom_min = base_scale * 0.75
+        zoom_max = base_scale * 1.35
+
+    zoom_period = 8.0  # seconds for a full in-out-in cycle
+
+    face_normals = [face_normal(f) for f in FACES]
```

Ruff (0.11.9)

Local variable `face_normals` is assigned to but never used

Remove assignment to unused variable `face_normals`

(F841)